### PR TITLE
Using Visio to control dashboard feed components

### PR DESCRIPTION
**Pull request by spetebroodt**

### CODE REVIEW

**Review by EdithLansens — APPROVED** [review]

File: user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+1. From the *Data* pane on the right, add a protocol filter:
+
+   1. Expand the *PROTOCOLS* section.
+
+   1. Search for the *Skyline Generic Virtual Connector*.
+
+   1. Drag the protocol onto the *Parameter feed* component to be used as a filter.
+
+   ![Protocols](~/user-guide/images/VisioURLFeed_ProtocolDataForFeed.png)
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md:96]: Could you add a better screenshot here that really shows how you add this as a filter?

File: user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+   ![Labels](~/user-guide/images/VisioURLFeed_StateLayoutOptions.png)
+
+1. Click *Stop editing* to exit edit mode.
+
+   At this point, you should be able to make manual parameter feed selections to control which parameter values are shown on the *State* component.
+
+   ![State component](~/user-guide/images/VisioURLFeed_DashboardManualResults.png)
+
+1. In the top-right corner of the screen, click the ellipsis button, and then click *Share*.
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md:122]: Could you add a screenshot of this action as well?

**Review by MariekeGO — CHANGES_REQUESTED** [review]

File: user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md (hunk 1 above)
Comment (spetebroodt) [reply user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md:96]: Done

File: user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md (hunk 2 above)
Comment (spetebroodt) [reply user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md:122]: Done

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**